README.md

```diff
@@ -19,13 +19,26 @@ Zeebe provides visibility into and control over business processes that span mul
 
 ## Release Lifecycle
 
-Our release cadence within major releases is a minor release every three months, with an alpha release on each of the two months between minor releases. Releases happen on the second Tuesday of the month, Berlin time (CET).
+Our release cadence within major releases is a minor release every six months, with an alpha release on each of the five months between minor releases. Releases happen on the second Tuesday of the month, Berlin time (CET).
 
-Minor releases are supported with patches for six months after their release.
+Minor releases are supported with patches for twelve months after their release.
 
-Here is a diagram illustrating the lifecycle of minor releases over a 13-month period:
+Here is a diagram illustrating the lifecycle of minor releases over a 20-month period:
```

Comment:
Please note, I chose a 20-month period, because it fits nicely on a page (under 80 chars).